Cargo.dev.toml

```diff
@@ -37,70 +37,1336 @@ resolver = "2"
+bp-header-chain = { git = "https://github.com/moonbeam-foundation//polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0" }
+bp-runtime = { git = "https://github.com/moonbeam-foundation//polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0" }
+frame-support = { git = "https://github.com/moonbeam-foundation//polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0" }
+frame-support-procedural = { git = "https://github.com/moonbeam-foundation//polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0" }
+frame-support-procedural-tools = { git = "https://github.com/moonbeam-foundation//polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0" }
+frame-support-procedural-tools-derive = { git = "https://github.com/moonbeam-foundation//polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0" }
+sp-api = { git = "https://github.com/moonbeam-foundation//polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0" }
+sp-api-proc-macro = { git = "https://github.com/moonbeam-foundation//polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0" }
+sp-core = { git = "https://github.com/moonbeam-foundation//polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0" }
+sp-core-hashing = { git = "https://github.com/moonbeam-foundation//polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0" }
+sp-debug-derive = { git = "https://github.com/moonbeam-foundation//polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0" }
+sp-externalities = { git = "https://github.com/moonbeam-foundation//polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0" }
+sp-std = { git = "https://github.com/moonbeam-foundation//polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0" }
+sp-storage = { git = "https://github.com/moonbeam-foundation//polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0" }
+sp-runtime-interface = { git = "https://github.com/moonbeam-foundation//polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0" }
+sp-runtime-interface-proc-macro = { git = "https://github.com/moonbeam-foundation//polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0" }
+sp-tracing = { git = "https://github.com/moonbeam-foundation//polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0" }
+sp-wasm-interface = { git = "https://github.com/moonbeam-foundation//polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0" }
+sp-io = { git = "https://github.com/moonbeam-foundation//polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0" }
+sp-keystore = { git = "https://github.com/moonbeam-foundation//polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0" }
+sp-state-machine = { git = "https://github.com/moonbeam-foundation//polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0" }
+sp-panic-handler = { git = "https://github.com/moonbeam-foundation//polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0" }
+sp-trie = { git = "https://github.com/moonbeam-foundation//polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0" }
```

Comment:
It is intentional as this branch includes moonbeam specific commits unlike moonbeam-polladot-v1.1.0. It will be changed only after the changes in the polkadot-sdk merged.